lib/internal/event_target.js

```diff
@@ -87,6 +87,14 @@ function isEvent(value) {
 }
 
 class Event {
+  /**
+   * @param {string} type
+   * @param {{
+   *   bubbles?: boolean,
+   *   cancelable?: boolean,
+   *   composed?: boolean,
+   * }} options
```

Comment:
Should we specify it as an optional parameter? I think that would be this?:

```suggestion
   * }} [options]
```

Reply:
Or maybe even better, specify it as optional and show the default value?:

```suggestion
   * }} [options=null]
```

I'm not 100% sure that's the right syntax. I came up with that by modeling the examples at https://jsdoc.app/tags-param.html#optional-parameters-and-default-values.